pkg/webui/ui/src/models.ts

```diff
@@ -236,6 +236,14 @@ export class VarsSourceVault {
         this.path = source["path"];
     }
 }
+export class VarsSourceGcpSecretsManager {
```

Comment:
can you also add GCP support here: https://github.com/kluctl/kluctl/blob/main/pkg/webui/ui/src/components/command-result/nodes/VarsSourceNode.tsx#L141

there is fallback code for unknown vars sources, but it would look a bit ugly :)